service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/RespondentControllerTest.java

```diff
@@ -43,19 +44,11 @@ void aboutToStartShouldPrepopulateRespondent() {
     @Test
     void shouldReturnDateOfBirthErrorsForRespondentWhenFutureDateOfBirth() {
         CaseDetails caseDetails = CaseDetails.builder()
-            .id(12345L)
-            .data(ImmutableMap.of(
-                "respondents1", ImmutableList.of(
-                    ImmutableMap.of(
-                        "id", "",
-                        "value", Respondent.builder()
-                            .party(RespondentParty.builder()
-                                .dateOfBirth(LocalDate.now().plusDays(1))
-                                .build())
-                            .build()
-                    )
-                )
-            ))
+            .data(Map.of("respondents1", wrapElements(Respondent.builder()
+                .party(RespondentParty.builder()
```

Comment:
we use this code a lot in this test class. would it be worth adding it to a private method rather than repeating it so much?

Respondent.builder()
.party(RespondentParty.builder()
.dateOfBirth(LocalDate.now().plusDays(1))
.build())
.build())))

Reply:
probably not going to do this, I am not too fussed about improving tests in this PR

Reply:
cool, happy with this :)